language/stdlib/modules/TravelRuleLimit.move

```diff
@@ -0,0 +1,39 @@
+address 0x0 {
+
+    module TravelRuleLimit {
+        use 0x0::LibraConfig;
+        use 0x0::Signer;
+        use 0x0::Transaction;
+        use 0x0::Association;
+
+        struct TravelRuleLimit {
+            micro_libra_limit: u64,
+        }
+
+        /// Travel rule limit set during genesis
+        public fun initialize(account: &signer) {
+            Transaction::assert(Signer::address_of(account) == LibraConfig::default_config_address(), 1);
+
+            LibraConfig::publish_new_config<TravelRuleLimit>(
+                account,
+                TravelRuleLimit { micro_libra_limit: 1000000 },
+            );
+        }
+
+        // Anyone can fetch the current set limit
+        public fun get_cur_microlibra_limit(): u64 {
+            let rule = LibraConfig::get<TravelRuleLimit>();
+            rule.micro_libra_limit
+        }
+
+        public fun set(account: &signer, micro_libra_limit: u64) {
+            Transaction::assert(Signer::address_of(account) == Association::treasury_compliance_account(), 3);
+
+            LibraConfig::set<TravelRuleLimit>(
```

Comment:
I'm very confused how this is working (assuming the functional test passes)...

`LibraConfig::set` requires that `account` must have a `TravelRuleLimit` config published under  it. However, `initialize` is publishing it under `default_config_address()`, and  `account` here is `0xB1E55ED`. You might need to store the `ModifyConfigCapability` from `publish_new_config_with_capability` in an internal (wrapper) resource, and then use that here to modify the config (using `set_with_capability`).